enterprise/frontend/src/metabase-enterprise/audit_app/lib/mode.js

```diff
@@ -76,8 +76,10 @@ const AuditDrill = ({ question, clicked }) => {
           name: "detail",
           title: `View this`,
           default: true,
-          url() {
-            return `/admin/audit/query/${encodeURIComponent(String(value))}`;
+          action() {
+            return push(
+              `/admin/audit/query/${encodeURIComponent(String(value))}`,
+            );
```

Comment:
i think forcing a page load was a hack to keep this page on life support, it wasn't working, so I changed it to use react-router which greatly improved performance.